lib/paths/api.go

```diff
@@ -1,7 +1,8 @@
 package paths
 
 const (
-	ReceiveAuthDocument = "/receiveAuthDocument"
-	SendAuthDocument    = "/sendAuthDocument"
-	ShowAuthToken       = "/showAuthToken"
+	ReceiveAuthDocument           = "/receiveAuthDocument"
+	RequestAwsRoleCertificatePath = "/aws/requestRoleCertificate"
```

Comment:
should we add a /v1 section to the path? so that we can version the api?

Reply:
Done.